readthedocs/search/serializers.py

```diff
@@ -90,20 +127,16 @@ def get_blocks(self, obj):
 
 class DomainHighlightSerializer(serializers.Serializer):
 
-    """
-    Serializer for domain results.
+    name = serializers.SerializerMethodField()
+    content = serializers.SerializerMethodField()
 
-    .. note::
+    def get_name(self, obj):
+        name = getattr(obj, 'domains.name', [])
+        return list(name)
 
-       We override the `to_representation` method instead of declaring each field
-       because serializers don't play nice with keys that include `.`.
-    """
-
-    def to_representation(self, instance):
-        return {
-            'name': getattr(instance, 'domains.name', []),
-            'docstring': getattr(instance, 'domains.docstrings', []),
-        }
+    def get_content(self, obj):
+        docstring = getattr(obj, 'domains.docstrings', [])
```

Comment:
Are we not changing the data model here to not have to reindex yet? Should there be a `TODO` mentioning updating the `docstrings` name on the model?

Reply:
Actually there is a note at the top of this file, since there are a lot of fields that should be renamed...